pubsub/pstest/fake.go

```diff
@@ -1102,12 +1102,14 @@ func (s *subscription) maintainMessages(now time.Time) {
 
 func (s *subscription) newStream(gs pb.Subscriber_StreamingPullServer, timeout time.Duration) *stream {
 	st := &stream{
-		sub:        s,
-		done:       make(chan struct{}),
-		msgc:       make(chan *pb.ReceivedMessage),
-		gstream:    gs,
-		ackTimeout: s.ackTimeout,
-		timeout:    timeout,
+		sub:               s,
+		done:              make(chan struct{}),
+		msgc:              make(chan *pb.ReceivedMessage),
+		gstream:           gs,
+		ackTimeout:        s.ackTimeout,
+		timeout:           timeout,
+		enableExactlyOnce: s.proto.EnableExactlyOnceDelivery,
+		enableOrdering:    s.proto.EnableMessageOrdering,
```

Comment:
Is `enableOrdering` here just to complete the fake?

Reply:
Yeah it's currently not used yet but I figured we could store it for now anyway. I'll add a TODO to explain this